blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -32,6 +32,7 @@
     QUERY_MODULES(""),
     DISCOVERY_MODULES(""),
     EVENT_LISTENER_MODULES(""),
+    EVENTS_MODULES(""),
```

Comment:
TODO: in blueflood-chef, we need to update site-cookbooks such that EVENTS_MODULES has the value of EventsElasticSearchIO